dev/devicelab/bin/tasks/plugin_dependencies_test.dart

```diff
@@ -274,13 +274,20 @@ public class DummyPluginAClass {
         section('Build plugin A example iOS app');
 
         await inDirectory(exampleApp, () async {
-          await evalFlutter(
+          final String output = await evalFlutter(
```

Comment:
I'm not sure this is the best place to put it. Since it's a test to check that interdependencies in plugins work correctly rather than checking the project->plugin connection. Should this just be in plugin_tests.dart?

Also how do we know plugins A/B/C/D says iOS 8? Tangentially, do we need to update https://github.com/flutter/flutter/blob/6b049372927f21f3d0de6d08d67545ae130c4a35/packages/flutter_tools/templates/plugin/ios-swift.tmpl/projectName.podspec.tmpl#L18 as well?

Reply:
> Should this just be in plugin_tests.dart?

I moved it there, see if you like it better.

> Also how do we know plugins A/B/C/D says iOS 8?

I changed the test to make the plugin version low (7.0) so it would trigger the warning on the devicelab machines running Xcode 11.

> Tangentially, do we need to update https://github.com/flutter/flutter/blob/6b049372927f21f3d0de6d08d67545ae130c4a35/packages/flutter_tools/templates/plugin/ios-swift.tmpl/projectName.podspec.tmpl#L18 as well?

If we updated to `s.platform = :ios, '9.0'` then newly created plugins would not be usable (complication error) in any projects created before https://github.com/flutter/flutter/issues/60035, and we have no migration (yet) to increase this value in already existing apps.